La'Tonia/key_logger_practice.py

```diff
@@ -0,0 +1,31 @@
+'''
+key logger project to create key logger to store key presses in text files
+
+project courtesy https://towardsdatascience.com/10-python-mini-projects-that-everyone-should-build-with-code-769c6f1af0c4
+'''
+
+# installed pynput using pip3 install pynput
```

Comment:
Are you only able to install with pip3? Might be mac thing but pip should work fine in a python3 environment